### ISSUE TITLE
Do I know you from somewhere?

### ISSUE THREAD

**Issue by rjhunjhunwala** [AnaMakarevich/anamakarevich.github.io#1]
Hey, I was just curious as to what had you discover my profile. Are you an intern at Barios?

**Comment by AnaMakarevich** [comment]
No (I don't even know what it is). I don't remember how exactly I discovered your profile. I thought it was quite cool and interesting to explore, that's it. It's not a social network after all. :)

**Comment by rjhunjhunwala** [comment]
Ah, cool. What's the paper's discussion "club" like. Is it accessible without too much deep ML background?

**Comment by AnaMakarevich** [comment]
Yes, it's totally accessible. That's the point of the club. It has just started. I am a part of Udacity and Facebook's challenge course on secure and private AI. And it turned out that there were a lot of people interested in the same thing :) So I organised a club. :) So far, some of the discussions are happening on Udacity's channel (it's by invite only), but everything else is available for everyone.
